backend/dataall/base/cdkproxy/requirements.txt

```diff
@@ -1,4 +1,5 @@
 aws-cdk-lib==2.83.1
+aws_cdk.aws_redshift_alpha==2.14.0a0
```

Comment:
where does this come from?

Reply:
It came from the main. But since we no longer have redshift, I will delete it. Thanks!